CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.17.0] - 2021-10-xx
+## [0.17.0] - 2021-11-03
 
```

Comment:
We need to add the RC tag changes here as well. Once you do so, I will go over them in one pass and suggest edits to make the changelog entries clearer.

Reply:
No.
RC is pre-release tags.

Reply:
As we do not recommend use dev, alpha, beta, rc marked packages in production - it is not a release.